setup.py

```diff
@@ -263,6 +263,9 @@ def use_bundled_lua(path, macros):
 
     # Parse .o files from 'makefile'
     makefile = os.path.join(path, "makefile")
+    if 'lua51' in path:
+        # Lua 5.1.5 source code is downloaded from lua.org/ftp
+        makefile = os.path.join(path, "src", "Makefile")
```

Comment:
Can't we have a list of places where we look, and take the first file that we find? I don't think "lua51" is special enough to break the rules here. And it might even fail to do the right thing if users name their working/venv directory "lua51_52_53", for example (because then "lua51" is in *every* path).

Reply:
How about checking the `libname` variable:

```python
if libname == 'lua51':
```